x-pack/plugins/ml/public/application/util/index_utils.ts

```diff
@@ -27,11 +23,11 @@ export async function getDataViewNames() {
  * @param job - Optional job object.
  * @returns The data view ID or null if it doesn't exist.
  */
-export async function getDataViewIdFromName(name: string, job?: Job): Promise<string | null> {
-  const dataViewsService = getDataViews();
-  if (dataViewsService === null) {
-    throw new Error('Data views are not initialized!');
-  }
+export async function getDataViewIdFromName(
```

Comment:
Would it be worth putting these functions in a provider which takes the `dataViewsService` ?
I see there are a few places where this is called multiple times in the same file.

Reply:
@walterra, @jgowdyelastic - this is already done via `indexServiceFactory` in `ml/public/application/util/index_service.ts` - this file will eventually replace this utils file.

Reply:
Luckily all remaining usages of this function were in places where we could make the new service easily available so I was able to get rid of this stand-alone function in [70ab770](https://github.com/elastic/kibana/pull/176788/commits/70ab770cc5e06985dba0dd7b86123eee21f7a72e). 🥳